### PR TITLE
Fixed no attribute 'ifelse'

### PR DESCRIPTION
**Pull request by slon1024**
Imported `ifelse` explicitly... without this it raised the exception `"theano.ifelse.ifelse" keras no attribute 'ifelse'` in this [line](https://github.com/fchollet/keras/blob/master/keras/backend/theano_backend.py#L1373)

The exception was raising for example for `Dropout` or `BatchNormalization` layers.

### CODE REVIEW

**Comment by KOLANICH** [comment]
That line was removed in 788c56b. Please re-rebase.
